lib/bbc/a11y/js/standards/exactlyOneMainLandmark.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  name: 'Exactly one main landmark',
+
+  validate: function($, fail) {
+    var mainLandmarks = $("[role='main']");
+    var count = mainLandmarks.length;
+    if (count == 0) {
+      fail('Found 0 elements with role="main".');
+    } else if (count > 1) {
```

Comment:
why two if statements here, why not do

if (count != 1) {
fail('Found ' + count + ' elements with role="main":', mainLandmarks);
}